clickhouse_db_schema/oss_ci_utilization/oss_ci_time_series_schema.sql

```diff
@@ -0,0 +1,35 @@
+-- This query creates the oss_ci_time_series table on ClickHouse
+CREATE TABLE misc.oss_ci_time_series(
+     -- created_at DateTime when the record is processed in db.
+    `created_at` DateTime64(0,'UTC'),
+    -- type of time series, for instance, utilization log data is 'utilization'.
+    `type` String,
+    `tags` Array(String) DEFAULT [],
+    `time_stamp` DateTime64(0,'UTC'),
```

Comment:
Note that there's also some potential to optimize sequential data like timestamps.  However, I'm not sure if it actually counts as sequential if the table is ordered by a different key.  

More info here: https://clickhouse.com/blog/working-with-time-series-data-and-functions-ClickHouse#codecs-to-optimize-sequences-storage